Packs/GCP-Enrichment-Remediation/Playbooks/playbook-Cloud_Response_-_GCP.yml

```diff
@@ -0,0 +1,2049 @@
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "10":
+    id: "10"
+    taskid: 35a2a97e-51fc-41a7-88d6-24c7aa59c52c
+    type: condition
+    task:
+      id: 35a2a97e-51fc-41a7-88d6-24c7aa59c52c
+      version: -1
+      name: Is there an access key to remediate?
+      description: Checks if there is an access key available for remediation.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "16"
+      "yes":
+      - "11"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.accessKeyName
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -750,
+          "y": 670
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "11":
+    id: "11"
+    taskid: 0102ae72-ebe0-44ac-844f-5844d224320a
+    type: condition
+    task:
+      id: 0102ae72-ebe0-44ac-844f-5844d224320a
+      version: -1
+      name: Should remediate automatically?
+      description: Checks if the user chose automatic remediation.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "39"
+      "yes":
+      - "13"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.autoAccessKeyRemediation
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -750,
+          "y": 860
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "12":
+    id: "12"
+    taskid: df4a0a5a-9ef2-4e1f-88eb-3859bc2f932b
+    type: regular
+    task:
+      id: df4a0a5a-9ef2-4e1f-88eb-3859bc2f932b
+      version: -1
+      name: Disable the access key
+      description: Disables a service account key.
+      script: '|||gcp-iam-service-account-key-disable'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "16"
+    scriptarguments:
+      key_name:
+        complex:
+          root: inputs.accessKeyName
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -750,
+          "y": 1660
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
```

Comment:
If we are not checking if the integration is enabled before the test, let's change it to true (let's verify it across all the response playbooks

Reply:
fixed